Problems/prob116/models/Vellino.essence

```diff
@@ -27,7 +27,7 @@ letting Bin be new type of size (sum m : Material . quantity(m))
 $ colour: each bin is assigned a colour
 $ contents: the contents of each bin is a multiset of materials
 find colour : function Bin --> Colour,
-     contents : function Bin --> mset (maxOccur maxMaterial) of Material
+     contents : function Bin --> mset (maxOccur maxMaterial, maxSize max(range(capacity))) of Material
```

Comment:
Same as Rack Configuration problem (`capacity` function is given, `contents` function returns an `mset` which has a constraint on its size), but with the following constraint:

```essence
$ the amount of material in each bin does not exceed its capacity
     forAll b in defined(colour) . |contents(b)| <= capacity(colour(b)),
```